common/src/stack/report-system/command/report/system/tests/test_backend_link.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import paramiko
+import socket
+from stack import api
+
+@pytest.mark.parametrize('backend',
+[host['host'] for host in  api.Call('list host', args=['a:backend'])]
+)
+class TestLinkUp:
+
+	# See if the interfaces stacki configures are actually up on the hosts
```

Comment:
I recommend using doc strings instead of a comment in the first line of a class. So:
```
class TestLinkUp:
	"""See if the interfaces stacki configures are actually up on the hosts"""
```